### ISSUE TITLE
Liaison to AG WG unclear

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/dpubwg-charter#53]
The liaison to AG WG is unclear to me, it's a lot of words that don't really translate into a coherent meaning that I can parse. I'm not quite sure what you're aiming at, but based on previous conversations I suggest something like "Address digital publishing use cases in accessibility guidelines, and develop techniques to implement accessibility features in e-books." (Or whatever the current politically correct synonym is for "e-books", I seem to keep mis-remembering which one is in vogue and don't mean to use the wrong term there.)

**Comment by avneeshsingh** [comment]
The statements are from high level view because it was created for the scope of the charter. We can make it more specific for AG working group by rephrasing it as follows:
The Publishing Working Group will coordinate with the AG Working Group to integrate accessibility requirements created as part of its recommendation-track deliverables into AG success criteria and techniques. One or more pipelines of the requirements will be maintained by Publishing Working Group to manage diverse turnaround times of the W3C groups.

**Comment by michael-n-cooper** [comment]
Hmm. I don't see that the proposed rewording is any different from what is in the charter draft, which prompted me to file this comment (or to say "I don't get it" to Ivan, who asked me to file it as a comment). I still find the wording difficult, I really don't know what it's proposing, and whether it's at all in line with my expectations of DPub and AG coordination. That said, the fact of the liaison is the most important thing, so I'm not gonna push too hard on the details. But if I were an AC reviewer I would file a "huh?" comment on this, so was trying to help out with a wording to avoid that.

**Comment by mattgarrish** [comment]
The second sentence is a tough parse. I believe it's simply acknowledging that we have different timelines and so things will happen at different speeds in different places (e.g., we might set a practice first, but then remove it later if it finds a general home in the AG in a future release). Does that really need saying, though, or is it an accepted fact?

Could we perhaps instead integrate Michael's prose and instead maybe draw parallels to what we say for CSS:

> The Publishing Working Group will work with the AG Working Group to address digital publishing use cases in accessibility guidelines, and develop techniques to implement accessibility features in web publications. The group will designate liaisons to work with the AG on issues as needed. Liaisons will ideally be members of both groups.

**Comment by michael-n-cooper** [comment]
That proposal works for me.

**Comment by iherman** [comment]
@avneeshsingh, is it o.k. with you? I would then make the changes and close this issue.

**Comment by pbelfanti** [comment]
+1 on Matt Garish's wording

**Comment by rickj** [comment]
+1 on @mattgarrish wording in https://github.com/w3c/dpubwg-charter/issues/53#issuecomment-291631834

**Comment by avneeshsingh** [comment]
Some clarifications and recommendation:
As I stated earlier, the statement was originally part of the scope, but it went through split, and then went through changes and now the meaning is not clear.
We had 3 objectives for placing all this in scope:
1. Publishing working group is responsible for identifying accessibility 
requirements that cannot be addressed by current WCAG versions.
2. Publishing working group need to develop and maintain accessibility requirements for digital publishing independent of WCAG life cycles so as to act as a consistent reference for the content producers.
3. Publishing working group need to coordinate with WCAG to incorporate these requirements into WCAG whenever possible.

So, if we change the scope statement a little, then the changes suggested by Michael and Matt will convey the complete message.
Current scope statement:
Recommendation-track deliverables will contain mechanisms to make 
Web Publications accessible to a broad range of readers with different needs 
and capabilities. This includes requirements of W3C WCAG 2.0 or ATAG 2.0 (or 
later versions) as well as requirements for international readers using 
different scripts and document formats. Additional extended requirements 
will be identified as conformance requirements in the Working Group’s 
normative specifications. Profiles of Web Publications may be defined with 
more stringent accessibility requirements.

We can extend the following sentence a little, to clarify the intention:
"Additional extended requirements will be identified as conformance 
requirements in the Working Group’s normative specifications, which will 
serve as a reference for content producers and for incorporation of 
publishing requirements in W3C standards like WCAG."

This minor change in scope statement will clarify the intent, and allow us to keep coordination section focused only on WCAG SC and techniques, as mentioned by Michael and matt.

**Comment by iherman** [comment]
@rickj @GarthConboy ?

**Comment by iherman** [comment]
It works for me.

**Comment by rickj** [comment]
I'm comfortable with this @iherman

**Comment by murata2makoto** [comment]
I am wondering who is responsible for the maintenance of EPUB accessibility 1.0, which is a part of EPUB 3.1.  Is it in the scope of the EPUB community group, AG WG, or the Publishing Working Group?

**Comment by mattgarrish** [comment]
EPUB specs will be maintained by the EPUB CG. Webpub accessibility will be a mix of the WG and feeding requirements to the AG WG.

**Comment by avneeshsingh** [comment]
George and I have the task to propose the future of EPUB 3 accessibility specs. to Publishing Business Group steering committee. Till now, the most practical way looks as to keep EPUB 3 related things in CG. I think that it is a different discussion topic from this issue, so may be it can be taken up in business group or IG conference calls independent of this GitHub issue.

**Comment by iherman** [comment]
@avneeshsingh @rickj @GarthConboy : I have merged the last pull request, covering this issue, too. @avneeshsingh, to be on the safe side, please check the scope section; if you are fine with it then this issue can be closed.

**Comment by avneeshsingh** [comment]
I have seen the diff, it looks fine. You can close this issue.
